### PR TITLE
Reverses the Yes/No buttons on play internet sound

### PR DESCRIPTION
**Pull request by Cyberboss**
So damn inconsistent

### CODE REVIEW

**Comment by PKPenguin321** [comment]
replace all yes and no buttons with "maybe" and "i dont know" buttons

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	admin_sound.status = SOUND_STREAM
 	admin_sound.volume = vol
 
-	var/res = alert(usr, "Show the title of this song to the players?",, "No", "Yes", "Cancel")
+	var/res = alert(usr, "Show the title of this song to the players?",, "Yes","No", "Cancel")
```
Comment (DaedalusGame) [inline code/modules/admin/verbs/playsound.dm]: you should all be ashamed of this line of code tbh

Comment (ExcessiveUseOfCobblestone) [reply code/modules/admin/verbs/playsound.dm]: Scared?

**Comment by SpaceManiac** [comment]
Fix it on line 103 too

File: code/modules/admin/verbs/playsound.dm (hunk 1 above)
Comment (DaedalusGame) [reply code/modules/admin/verbs/playsound.dm]: wrong

**Comment by Dorsisdwarf** [comment]
isnt this so that the auto-selected option is "no" since you usually dont want to show the song name to players?

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**